quirks/password-rules.json

```diff
@@ -422,6 +422,9 @@
     "leetchi.com": {
         "password-rules": "minlength: 8; required: lower; required: upper; required: digit; required: [!#$%&()*+,./:;<>?@\"_];"
     },
+    "lepida.it": {
+        "password-rules": "minlength: 8; maxlength: 16; required: lower; required: upper; required: digit; required: special;"  
```

Comment:
Hi @guidev 

Thanks for taking the time to create this PR :)

If I understand rule (e) correctly, the website seems to have a `max-consecutive` requirement of 2 characters. Please include that in your password rules string.

Also, if possible, it would also be nice to verify the exact characters they recognize as special characters.